.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,51 +1,41 @@
-- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
-- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
-  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
-- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
-  - [ ] (Module updates) Added a release notes entry if the change is significant
-  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
-- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md), [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) and other contributing documentation in corresponding paths.
-
-<!--
-To help with the large amounts of pull requests, we would appreciate your
-reviews of other pull requests, especially simple package updates. Just leave a
-comment describing what you have tested in the relevant package/service.
-Reviewing helps to reduce the average time-to-merge for everyone.
-Thanks a lot if you do!
-
-List of open PRs: https://github.com/NixOS/nixpkgs/pulls
-Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
--->
-
----
-
-Add a :+1: [reaction] to [pull requests you find important].
-
-[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
-[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc
+- Tested:
+  - [ ] [NixOS test][1] in [nixos/tests][2]
+  - [ ] [package tests][3] at `passthru.tests`
+  - [ ] tests in [lib/tests][4] or [pkgs/test][5] for functions and "core" functionality
+- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage][7].
+- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
+- [Nixpkgs 25.11 Release Notes][nixpkgs-25.11]
+  - [ ] Package update: when the change is major or breaking.
+- [NixOS 25.11 Release Notes][nixos-25.11]
+  - [ ] Module addition: when adding a new NixOS module.
+  - [ ] Module update: when the change is significant.
+- [ ] Fits [CONTRIBUTING.md][A], [pkgs/README.md][B], [maintainers/README.md][C] and others READMEs.
+
+[1]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
```

Comment:
Thoughts on using named links instead of numbered?

IMO numbered links should usually follow the order they appear in throughout the document, which may change over time if links are added or removed. This is less important with named links.

Additionally, it's easier to cross-reference named links and remember which one is which.

I appreciate it is hard to balance brevity with uniqueness, though. E.g. naming this `[nixos-sec-nixos-tests]` is probably ok, but it's definitely more verbose than `[1]`.